test/Ocelot.UnitTests/DependencyInjection/ConfigurationBuilderExtensionsTests.cs

```diff
@@ -321,7 +321,7 @@ private void GivenTheBaseUrl(string baseUrl)
 
         private void WhenIGet(string key)
         {
-            _result = _configuration.GetValue("BaseUrl", "");
+            _result = _configuration.GetValue(key, "");
```

Comment:
Was this test broken? If so, why didn't we see this sooner? If not, what changed in the implementation that required this change?

Reply:
It was not broken. The caller always set the key to “BaseUrl” :)